builtin/logical/pki/fields.go

```diff
@@ -22,6 +22,17 @@ key and issuing cert will be appended to the
 certificate pem. Defaults to "pem".`,
 	}
 
+	fields["private_key_format"] = &framework.FieldSchema{
```

Comment:
Add this param to the docs page as well.

Reply:
Done.